pkg/webui/components/input/index.js

```diff
@@ -78,10 +79,11 @@ class Input extends React.Component {
     component: 'input',
     disabled: false,
     error: false,
-    inputWidth: 'm',
+    inputWidth: undefined,
```

Comment:
Please add a comment that the default value is set programmatically based on input type.